src/test/kotlin/com/beust/kobalt/VerifyKobaltZipTest.kt

```diff
@@ -37,6 +38,11 @@ class VerifyKobaltZipTest : KobaltTest() {
                             throw KobaltException("kobaltw has wrong line endings")
                         }
                     }
+                    if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
Use `OperatingSystem.current().isWindows()` for this.

This class (and related classes) actually come straight from Gradle, they could probably use some clean up and kotlinization.